Description: these require a working `journalctl --user` in the test environment..

index c2f30108..78493210 100644
Index: tracing-journald/tests/journal.rs
===================================================================
--- tracing-journald.orig/tests/journal.rs
+++ tracing-journald/tests/journal.rs
@@ -166,6 +166,7 @@ fn retry_read_one_line_from_journal(test
     .unwrap()
 }
 
+#[ignore]
 #[test]
 fn simple_message() {
     with_journald(|| {
@@ -212,6 +213,7 @@ fn custom_priorities() {
     with_journald_layer(layer, test);
 }
 
+#[ignore]
 #[test]
 fn multiline_message() {
     with_journald(|| {
@@ -223,6 +225,7 @@ fn multiline_message() {
     });
 }
 
+#[ignore]
 #[test]
 fn multiline_message_trailing_newline() {
     with_journald(|| {
@@ -237,6 +240,7 @@ fn multiline_message_trailing_newline()
     });
 }
 
+#[ignore]
 #[test]
 fn internal_null_byte() {
     with_journald(|| {
@@ -248,6 +252,7 @@ fn internal_null_byte() {
     });
 }
 
+#[ignore]
 #[test]
 fn large_message() {
     let large_string = "b".repeat(512_000);
@@ -263,6 +268,7 @@ fn large_message() {
     });
 }
 
+#[ignore]
 #[test]
 fn simple_metadata() {
     let sub = Layer::new()
@@ -304,6 +310,7 @@ fn journal_fields() {
     });
 }
 
+#[ignore]
 #[test]
 fn span_metadata() {
     with_journald(|| {
@@ -328,6 +335,7 @@ fn span_metadata() {
     });
 }
 
+#[ignore]
 #[test]
 fn multiple_spans_metadata() {
     with_journald(|| {
@@ -354,6 +362,7 @@ fn multiple_spans_metadata() {
     });
 }
 
+#[ignore]
 #[test]
 fn spans_field_collision() {
     with_journald(|| {
